docs/team-process.md

```diff
@@ -1,3 +1,5 @@
+<!-- how important is it to keep this file up to date? can we delete this file altogether? -->
```

Comment:
This file is super important IMO. It serves as living documentation of the team working styles and agreements as well as our dependency update procedure. I believe this is referred to and updated weekly!